components/edit.js

```diff
@@ -215,36 +219,70 @@ const EditPage = ({ item }) => {
           canSave: true,
           errors: [],
         });
-        const errors = response.errors.map((err) => ({
-          description: err.description,
-          keys: err.name
-            .map((val) => {
-              return `\`${val}\``;
+        const errors = response.errors
+          ? response.errors.map((err) => {
+              // Surround each err name with ``, and separate by comma
```

Comment:
I mean they don't *need* it, but I just thought it looked nice to kind of highlight the names. Like, if this were markdown, the backticks would render each name with a monospace font. I could use just regular/single quotes instead I guess?

Reply:
OK, I’m surprised that even works. I would have figured it just displays the backticks, not formatted code-style text.

Reply:
oh yeah, it does just display the backticks. But, whenever I want to communicate "this is code-like" if the text won't be rendered like markdown, I'll type in backticks because of the markdown standard.

Reply:
Oh, I think if it were me, I would format the text in some way. Probably the easiest is just to pass this text with the backticks to the marked package we have so it’ll actually display formatted. I feel like the backticks aren’t really for presentation. I use them in code comments, though VSCode often displays those formatted in Intellisense.